python/constraints.txt

```diff
@@ -425,10 +427,12 @@ pyarrow==14.0.2
     #   streamlit
 pycparser==2.21
     # via cffi
-pydantic==1.10.13
+pydantic==2.5.3
```

Comment:
@damonmcc - that was quick

Reply:
they did it!

Reply:
Also for context if others are looking - we downgraded like a week ago because latest dbt required pydantic 1